apps/dashboard/src/DashboardApp.vue

```diff
@@ -39,7 +39,7 @@
 							{{ t('dashboard', '"{title} icon"', { title: apiWidgets[panels[panelId].id].title }) }}
 						</span>
 					</div>
-					<div class="panel--content">
+					<div class="panel--content" :id="`panel-${panels[panelId].id}`">
```

Comment:
```suggestion
					<div class="panel--content">
```
Unnecessary addition left-over from before